frontend/packages/console-plugin-sdk/OWNERS

```diff
@@ -7,4 +7,4 @@ approvers:
   - spadgett
   - vojtechszocs
 labels:
-  - component/sdk
+  - component/sdk-static
```

Comment:
I say keep it as `component/sdk` and reuse the same label for other packages that are sdk related.

Reply:
OK, sounds good :+1: